gen/generator.go

```diff
@@ -849,7 +861,8 @@ func genAnnotationUnsupported() []string {
 }
 
 func main() {
-	getClassMetadata()
+	reGenClassMetadata()
+	getClassMetadata("")
```

Comment:
can we pass in this in here, then the conditional would not be needed in getClassMetadata?

```suggestion
	getClassMetadata(os.Getenv("GEN_CLASSES"))
```

Reply:
Done